templates/rsyslog.conf.erb

```diff
@@ -55,23 +55,9 @@ $ActionSendStreamDriverPermittedPeer <%= @permitted_peer %>
 $RuleSet local
 
 <% end -%>
-# Log all kernel messages to the console.
-# Logging much else clutters up the screen.
-#kern.*                                                 /dev/console
-kern.*                                                  <%= @kernel_target %>
-
-# Log anything (except mail) of level info or higher.
-# Don't log private authentication messages!
-*.info;mail.none;authpriv.none;cron.none                /var/log/messages
-
-# The authpriv file has restricted access.
-authpriv.*                                              /var/log/secure
-
-# Log all the mail messages in one place.
-mail.*                                                  -/var/log/maillog
-
-# Log cron stuff
-cron.*                                                  /var/log/cron
+<% @log_entries_real.each do |log_entry| -%>
+<%= log_entry %>
+<% end -%>
 
 # Everybody gets emergency messages
```

Comment:
hahahaha...
Here we call it Karneval and as kid I loved it for the costumes and the action and never forget the free candies. But as grown up I can't stand the music anymore ;)